src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --secondarycolor: #00acdc;
+  --primarycolor: #000;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 50px;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 15px 0;
+  color: var(--primarycolor);
+}
+
+.nav__link:hover {
+  color: var(--secondarycolor);
+}
+
+.nav__list {
```

Comment:
Please reset margin for the list because is your solution there is some margin top & bottom.